### DISCUSSION TITLE
ERROR EN COLAB CON ACCESO A MI CUENTA

### DISCUSSION THREAD

**Discussion by paulajimeneza** [rramosp/ai4eng.v1#107] [00 - General]
Hola buenas tardes, es que al validar mi cuenta en colab, hoy no me aparece mi cuenta asociada a la del curso, solo dice "See my course and progress", inicio sesión pero no aparece mi cuenta y por lo tanto no aparece mi calificación, espero me puedan ayudar, muchas gracias. Quedo atenta

**Comment by Felipe-RA** [comment]
Saludos paulajimeneza

Pongo por aquí una marca de que lo solucionamos en sesión sincrónica!

Hasta pronto
